ovos_persona_server/ollama.py

```
@@ -0,0 +1,435 @@
+
+    Returns:
+        Union[JSONResponse, StreamingResponse]: A JSON response for non-streaming
+                                                or a StreamingResponse for streaming requests.
+
+    Raises:
+        HTTPException: If the prompt is missing or persona generation fails.
+    """
+    prompt: str = request_body.prompt
+    stream: bool = request_body.stream
+    system: Optional[str] = request_body.system
+    suffix: Optional[str] = request_body.suffix
```

Ruff (0.12.2)

Local variable `suffix` is assigned to but never used

Remove assignment to unused variable `suffix`

(F841)